addons/l10n_nl/data/account_fiscal_position_template.xml

```diff
@@ -28,83 +28,42 @@
         <record id="position_tax_eu_no_taxes_report_1" model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fiscal_position_template_eu_no_taxes_report"/>
             <field name="tax_src_id" ref="btw_0"/>
+            <field name="tax_dest_id" ref="btw_X2"/>
         </record>
         <record id="position_tax_eu_no_taxes_report_2" model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fiscal_position_template_eu_no_taxes_report"/>
-            <field name="tax_src_id" ref="btw_6"/>
```

Comment:
Shouldn't the mapping of BTW 6% (`btw_6`) be kept instead of being replaced, and a new mapping BTW 9% (`btw_9`) be added?

Reply:
Hi, the fiscal position is only for sales to customers within EU. So no purchase.

Reply:
@ploegvde I mentioned `btw_6` and `btw_9`, both are for sales.
@Yenthe666 What is the reason for replacing `btw_6`? Shouldn't be kept instead?

Reply:
@ploegvde indeed, I think you misunderstood.
I removed the `btw_6` as 6% is no longer used and it is now `btw_9` (9%). All new schema's shouldn't be using 6% anymore and all the old ones are still correctly configured and will not be updated anymore. I believe that covers all cases.

Reply:
@Yenthe666 If you are entering old invoices with `btw_6` you probably need the related fiscal position mapping as well.

When you say:

> All new schema's shouldn't be using 6% anymore

you are assuming that tax `btw_6` should also not to be used anymore, but you kept it in the tax template for some valid reasons. Same reasons apply to the `btw_6` fiscal positions mapping, so it should be kept as well.

Reply:
It would be better to add the btw_6 as well as it is still there everywhere else.  (just consistency)